### PR TITLE
[FT]/Add a blog where users can share ideas

### PR DESCRIPTION
**Pull request by thoth-ky**
#### Problem
- There was  nothing on HealthChecks that allows the tech community to contribute knowledge and expertise on automating business systems

#### Solution
- I created a basic blog that allows logged in users to write blog posts and share them. A user is free to assign their blog categories and tags

#### Screenshots
**Home view of blogs**
blog_home

**Form for posting blogs**
blog_post
**Draft View**
draft_view
**Blog View**
blog view

[PT CARD](https://www.pivotaltracker.com/story/show/158532888)

### CODE REVIEW

**Review by dmaish — APPROVED** [review]
code looks and works awesome.good work.

**Review by miritih — APPROVED** [review]
Good work.

File: hc/blog/models.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
+from django.db import models
+from hc.accounts.models import User
+from django.utils import timezone
+from django.urls import reverse
+
+
+STATUS_CHOICES = (
+	('draft', 'Draft'),
+	('published', 'published'))
```
Comment (simiyu1) [inline hc/blog/models.py:9]: Good work, now if you could capitalise 'p' in `('published', 'Published')`

**Review by simiyu1 — APPROVED** [review]

**Comment by thoth-ky** [comment]
project archived

**Pull request closed without merging**